File: ext/internal/features/sleeper/payloads/payloads.go
// Package payloads holds the Go counterparts of the PHP Sleeper payload objects
// (SConcur\Features\Sleeper\Payloads\*). The struct tags are the short keys emitted by
// the PHP getData() methods.
package payloads

// SleeperPayload is the payload of a sleep command.
// PHP: SConcur\Features\Sleeper\Payloads\SleeperPayload.
type SleeperPayload struct {
	Microseconds int64 `json:"us" msgpack:"us"`
}

File: ext-rust/src/features/sleeper/payloads.rs
//! Mirrors ext/internal/features/sleeper/payloads: the Rust counterpart of the
//! PHP Sleeper payload objects. The field renames are the short keys emitted by
//! the PHP getData() methods.

use serde::Deserialize;

#[derive(Deserialize)]
pub struct SleeperPayload {
    #[serde(rename = "us")]
    pub microseconds: i64,
}
